src/main/java/org/folio/edge/sip2/repositories/PatronRepository.java

```diff
@@ -159,6 +159,16 @@ private Future<PatronInformationResponse> validPatron(String userId, Personal pe
     final Future<PatronInformationResponseBuilder> manualBlocksFuture = feeFinesRepository
         .getManualBlocksByUserId(userId, sessionData)
         .map(blocks -> buildPatronStatus(blocks, builder));
+    // add fine count
```

Comment:
Maybe capitalize "a" in the comment to make it conform to other comments in the code. It's super clean code! Might as well make the comments clean too.

Reply:
Added